.vscode/settings.json

```diff
@@ -1,4 +1,4 @@
 {
-  "typescript.tsdk": "node_modules/typescript/lib",
+  "typescript.tsdk": "node_modules\\typescript\\lib",
```

Comment:
we can probably get rid of this change

Reply:
@FrancesCoronel --For some reason, this same change keeps coming up every time I go to commit! I've went ahead and discarded it though!